### PR TITLE
Fix for multi account stack changes (for core service deployment)

### PR DESCRIPTION
**Pull request by SauarvD**
### Requirements

*Fixing deployment changes

### Description of the Change

Since for the core services, there are no deployment accounts field, we have to put a check and fetch the default data

### Benefits

Build won't fail during stack creation

### CODE REVIEW

File: builds/jazz-build-module/service-metadata-loader.groovy [hunk 1]
```diff
@@ -49,8 +49,13 @@ def loadServiceMetadata(service_id){
 			metadata['created_by'] = service_data.Item.SERVICE_CREATED_BY.S
 			metadata['type'] = service_data.Item.SERVICE_TYPE.S
 			metadata['runtime'] = service_data.Item.SERVICE_RUNTIME.S
-			metadata['accountId'] = service_data.Item.SERVICE_DEPLOYMENT_ACCOUNTS.L[0].M.accountId.S
-      metadata['region'] = service_data.Item.SERVICE_DEPLOYMENT_ACCOUNTS.L[0].M.region.S
+			if(service_data.Item.SERVICE_DEPLOYMENT_ACCOUNTS){
+			    metadata['accountId'] = service_data.Item.SERVICE_DEPLOYMENT_ACCOUNTS.L[0].M.accountId.S
+                metadata['region'] = service_data.Item.SERVICE_DEPLOYMENT_ACCOUNTS.L[0].M.region.S
```
Comment (supritat) [inline builds/jazz-build-module/service-metadata-loader.groovy]: Need indentation fix

File: builds/jazz-build-module/service-metadata-loader.groovy [hunk 2]
```diff
@@ -49,8 +49,13 @@ def loadServiceMetadata(service_id){
...
+			} else {
+			    metadata['accountId'] = configLoader.AWS.DEFAULTS.ACCOUNTID
+                metadata['region'] = configLoader.AWS.DEFAULTS.REGION
```
Comment (supritat) [inline builds/jazz-build-module/service-metadata-loader.groovy]: Need indentation fix

Comment (SauarvD) [reply builds/jazz-build-module/service-metadata-loader.groovy]: fixed it

File: builds/jazz-build-module/service-metadata-loader.groovy (hunk 1 above)
Comment (SauarvD) [reply builds/jazz-build-module/service-metadata-loader.groovy]: fixed it

**Pull request merged**